pkgs/top-level/all-packages.nix

```diff
@@ -8744,7 +8744,11 @@ in
 
   fish = callPackage ../shells/fish { };
 
-  fish-foreign-env = callPackage ../shells/fish/fish-foreign-env { };
+  fishPlugins = recurseIntoAttrs (callPackage ../misc/fish-plugins { });
+
+  fish-foreign-env = throw ''
```

Comment:
You should define an alias in `pkgs/top-level/aliases.nix`, otherwise this will break existing setups.

Reply:
Amended :+1: